Pairing up the NphPairedSite field for NPH Participant API

xtoller opened the pull request:
## Resolves *[ticket #](link to ticket)*


## Description of changes/additions
Pairing the NphPairedSite to NPH Participant API 

## Tests
- [] unit tests

xtoller committed:
Pairing up the NphPairedSite field for NPH Participant API

j-kanuch commented on rdr_service/api/nph_participant_api_schemas/schema.py:
```diff
@@ -252,10 +264,22 @@ class Meta:
+                                                                              PairingEvent.event_authored_time
+                                                                              < pm2.event_authored_time)
+                                                                    ).join(nphSite, nphSite.id == PairingEvent.site_id
+                                                                           ).filter(pm2.id.is_(None))
```
I hadn't thought about this at the time we talked about the query, but since this query is only to be ran on NPH, we probably want to add another filter condition here so that we don't need to update the query later when we add another ancillary study. It would look like:
.filter(ParticipantMapping.ancillary_study_id == 2)

xtoller replied:
included another filter as Josh suggested.

j-kanuch commented on rdr_service/api/nph_participant_api_schemas/schema.py:
```diff
@@ -188,6 +190,19 @@ class Participant(ObjectType):
+    organization_external_id = SortableField(String, name="nphPairedOrg", description='Sourced from NPH Schema.',
+                                             sort_modifier=lambda context: context.set_order_expression(
+                                                 nphSite.organization_external_id))
+    awardee_external_id = SortableField(String, name="nphPairedOrg", description='Sourced from NPH Schema.',
```
This should be something like `nphPairedAwardee` not `nphPairedOrg`

xtoller replied:
I'm being not careful about this one. already updated to show awardee_external_id to the name "nphPairedAwardee"

xtoller committed:
schema.py - modification per Josh's comment
util.py - including nph fields org_external_id & awardee_id
test_nph_participant_api.py - refactor to unit test add unit test for nph fields: org_external_id & awardee_id

xtoller commented:
Just added another commit to this PR:

schema.py - modification per Josh's comment
util.py - including nph fields org_external_id & awardee_id
test_nph_participant_api.py - refactor to unit test add unit test for nph fields: org_external_id & awardee_id

j-kanuch reviewed (approved):
Thanks for adding the filter. I've made a note to revisit it for a later refactor to make filtering for the studies a little cleaner.
